### DISCUSSION TITLE
Global Dark Mode Integration

### DISCUSSION THREAD

**Discussion by IdleEndeavor** [minbrowser/min#1472] [Ideas]
Sorry for typing a lot of suggestions.
It would be very nice if you could integrate something like the Dark Reader Extension into the browser or mimic the functionality of it. Dark Reader reformats websites so even if they don't have dark mode they can be made dark. The formatting is pretty good too, I almost never have to turn it off due to errors or bad formats.
I think it is opensource, kindly do try to integrate this.
Thanks a Lot!

**Comment by Syndamia** [comment]
There is already a userscript for that: https://github.com/PalmerAL/min-dark-mode-userscript

**Reply by IdleEndeavor** [reply]
Thanks a lot for the information and sorry for this post.
